### ISSUE TITLE
Redesign LogitLens input/prompt area

### ISSUE THREAD

**Issue by AdamBelfki3** [ndif-team/workbench#45]
The idea here would be to create a chat like interface with the prompt area in the bottom and all the selectors. For the line chart, add a search bar at the top to select token logits to visualize.
